tools/m1_utils/update_static.sh

```diff
@@ -47,7 +47,7 @@ patch() {
      ARCHS=()
      mkdir -p "$(dirname $OF)"
      while read ARCH; do
-          ARCHS+=($ARCH)
+          ARCHS+=("$ARCH")
```

Comment:
@sethfri this change would break the program because it currently it relies on wordsplitting in bash to split the variable into an array.

I think it's safe to read to an array too - does that pass the shellcheck config?
```
read -a ARCHS < <(lipo -archs "$FWF")
```

Reply:
Edit: _to fix that, you can replace the entire while loop with the above statement_ ^